src/libs/testing-library/utils/mock.ts

```diff
@@ -0,0 +1,120 @@
+import { debugTokens } from '../../../../e2e/utils/types';
+import { RequestHandler } from 'msw';
+import { setupServer } from 'msw/node';
+import { HttpResponse, http } from 'msw';
+import tokenListsMock from '../../../../e2e/mocks/tokenLists.json';
```

Comment:
Suggestion: If we use E2E files, maybe we could move the testing logic into a root folder : 
`tests/e2e`
`tests/integration`

Reply:
Not doing for now as got a lot of issues with imports in *.test.* files as well as using vi.mock when moving the testing-library to tests/integration